ppb/assetlib.py

```diff
@@ -165,6 +165,10 @@ class AbstractAsset(abc.ABC):
     This defines the common interface for virtual assets, proxy assets, and
     real/file assets.
     """
+    # Can be overridden on child classes to add an additional message to show after "File not
+    # found: %r.". See Image.not_found_message for an example.
```

Comment:
This should be #: so it gets picked up by sphinx

Reply:
I didn't know about that #: syntax, thanks for the pointer. I just read a bit about it and updated those two comment lines.